client/src/extension.ts

```diff
@@ -37,27 +37,25 @@ export function activate(context: vscode.ExtensionContext) {
         };
     } else {
         console.log('Starting without debug');
-        // Name of the launcher class which contains the main.
-        const main: string = 'org.drools.lsp.server.DroolsLspLauncher';
 
-        const {JAVA_HOME} = process.env;
+        const javaHome = getJavaHome();
 
-        // If java home is available continue.
-        if (JAVA_HOME) {
-            // If java home is available continue.
-            console.log(`Using java from JAVA_HOME: ${JAVA_HOME}`);
-            let excecutable: string = path.join(JAVA_HOME, 'bin', 'java');
+        let executable: string = `java`;
 
-            // path to the launcher.jar
-            let classPath = path.join(__dirname, '..', '..', 'drools-lsp-server', 'target', 'drools-lsp-server-jar-with-dependencies.jar');
-            const args: string[] = ['-cp', classPath];
-
-            serverOptions = {
-                command: excecutable,
-                args: [...args, main],
-                options: {}
-            };
+        if (javaHome) {
```

Comment:
Maybe consider showing a warning/error message to users if `javaHome` is `undefined`?

Reply:
@caponetto Thanks, I added the warn.